app/src/main/java/com/codepath/selfiespot/activities/TempDetailSelfieSpotActivity.java

```diff
@@ -200,8 +204,65 @@ public void done(final ParseException e) {
                 }
             }
         });
+
```

Comment:
think we don't need to constrain this, so, doesn't matter if you want to share multiple times, remove this